packages/addons/test/plugins-support.test.ts

```diff
@@ -15,76 +15,81 @@ limitations under the License.
 */
 
 import { describe, expect, test } from '@jest/globals';
-import { BpmnVisualization } from '../dist';
+import type { Plugin } from '../src';
+import { BpmnVisualization } from '../src';
+
+interface TestPlugin extends Plugin {
```

Comment:
**nitpick:** please remove the interface. I wrote the test and there was on intention for the 2 plugins to share any method because they are not related.
I'd prefer to change the method name in the 2nd plugin to show that there's no link with the 1st plugin.

Reply:
If we don't have any interface for the plugin or only the interface `Plugin`, we have errors like following:  
 

![](https://github.com/process-analytics/bv-experimental-add-ons/assets/4921914/dc3bc35e-e0a7-4c2f-93a9-6efea923c94a)